docs/core/tools/dotnet-install-script.md

```diff
@@ -106,91 +96,45 @@ Specifies the URL for the Azure feed to the installer. It isn't recommended that
 
 If set, the installer uses the proxy when making web requests.
```

Comment:
if the sections are really combined, then you should say this is only valid for Windows. Unless we're missing this in the syntax for macOS/Linux.

Reply:
done...